panacea/query_client_test.go

```diff
@@ -1,34 +1,28 @@
 package panacea_test
 
-import (
-	"context"
-	"encoding/hex"
-	"github.com/cosmos/cosmos-sdk/types/bech32"
-	"github.com/medibloc/panacea-doracle/panacea"
-	"github.com/stretchr/testify/require"
-	"testing"
-)
+// All the tests can only work in sgx environment, so the tests are commented out.
```

Comment:
Okay. I think it's okay to assume that this project works only in SGX (It means that you don't need to care about the local dev on MacOS). But, let's go with commenting this out for now. I'll reoragnize the whole testing strategy soon.